src/main/java/net/rptools/maptool/client/ui/campaignproperties/CampaignPropertiesDialog.java

```diff
@@ -529,4 +553,354 @@ public static void main(String[] args) {
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <th></th>
+              <th>${optionLabelRange}</th>
+              <td>${optionDescriptionRange}</td>
+              <td></td>
+              <td${alignCellCenter}>30</td>
+            </tr>
+            """;
+    String optionsSight =
+        """
+            <tr>
+              <th>${optionLabelDistance}</th>
+              <td${alignCellCenter}>${optionTypeKeyEqualsValue}</td>
+              <td>${optionDescriptionDistance}</td>
+              <td${alignCellCenter}>${mapVisionDistance}</td>
+              <td${alignCellCenter}>distance=120</td>
+            </tr>
+            <tr>
+              <th>${optionLabelScale}</th>
+              <td${alignCellCenter}>${optionTypeKeyword}</td>
+              <td>${optionDescriptionScale}</td>
+              <td${alignCellCenter}>${wordUnused}</td>
+              <td>scale</td>
+            </tr>
+            <tr>
+              <th>${optionLabelMagnifier}</th>
+              <td${alignCellCenter}>${optionTypePrefixedValue}</td>
+              <td><i>[ x0.0 ]</i> ${optionDescriptionMagnifier}</td>
+              <td${alignCellCenter}>x1</td>
+              <td${alignCellCenter}>x2.5</td>
+            </tr>
+            <tr>
+              <th>${optionLabelPersonalSight}</th>
+              <td${alignCellCenter}>${optionTypeSpecial}(${wordString})</td>
+              <td>${optionDescriptionPersonalSight} ${phraseMultipleEntriesAllowed}<sup>3</sup></td>
+              <td${alignCellCenter}>${wordUnused}</td>
+              <td${alignCellCenter}>r30#afafaa+100</td>
+            </tr>
+            <tr>
+              <th></th>
+              <th>${columnHeadingOptionComponent}</th>
+              <th>${wordSyntax}&nbsp;&#10233;&nbsp;r00|#rrggbb|+y&nbsp;&nbsp;(${optionLabelRange}|${optionLabelColor}|${optionLabelLumens})</th>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <th></th>
+              <th>${optionLabelRange}</th>
+              <td><i>[${optionTypePrefixedValue} "r"]</i> ${optionDescriptionPersonalSightComponentRange}</td>
+              <td></td>
+              <td${alignCellCenter}>r30</td>
+            </tr>
+            """;
+    String optionsCommon_2 =
+        """
+            <tr>
+              <th></th>
+              <th>${optionLabelColor}</th>
+              <td><i>[${wordOptional}]</i>&nbsp;${optionDescriptionPersonalSightComponentColor}</td>
+              <td></td>
+              <td${alignCellCenter}>#afafaa</td>
+            </tr>
+            <tr>
+              <th></th>
+              <th>${optionLabelLumens}</th>
+              <td><i>[${wordOptional}]</i>&nbsp;${optionDescriptionPersonalSightComponentLumens}<sup>4</sup></td>
+              <td${alignCellCenter}>+100</td>
+              <td${alignCellCenter}>+100</td>
+            </tr>
+            </table>
+            """;
```

Comment:
Same thing as above. If such blocks are necessary, have the opening and closing tags in separate strings (called blaOpen and blaClose). That removes the implicit order for the first and last text blocks.

Reply:
Easy done. Shall implement.